OnServiceChange/handler.ts

```diff
@@ -161,14 +172,22 @@ export const createUpsertSql = (dbConfig: IDecodableConfigPostgreSQL) => (
     .insert(data)
     .onConflict("id")
     .merge([
-      "organizationFiscalCode",
-      "version",
-      "name",
+      "authorizedCIDRS",
+      "departmentName",
+      "description",
       "isVisible",
+      "maxAllowedPaymentAmount",
+      "name",
+      "organizationFiscalCode",
+      "organizationName",
+      "quality",
+      "scope",
+      "serviceMetadata",
+      "subscriptionAccountEmail",
       "subscriptionAccountId",
       "subscriptionAccountName",
       "subscriptionAccountSurname",
-      "subscriptionAccountEmail"
+      "version"
     ])
     .whereRaw(`"${dbConfig.DB_TABLE}"."version" < excluded."version"`)
```

Comment:
Use `<=` so we can update records if we replay change feed
```suggestion
    .whereRaw(`"${dbConfig.DB_TABLE}"."version" <= excluded."version"`)
```

Reply:
Fix with 5530311